pkg/scheduler_patcher.go

```diff
@@ -32,11 +33,14 @@ const (
 	schedulerPath = "/etc/kubernetes/scheduler"
 	manifestsPath = "/etc/kubernetes/manifests"
 	configPath    = "/conf"
+
+	platformOpenshift = "openshift"
 )
 
 type SchedulerPatcher struct {
 	kubernetes.Clientset
 	logr.Logger
+	Client client.Client
```

Comment:
Why do we need `client.Client`, if we've already had `kubernetes.Clientset` here. Can we use it instead of client.Client?

Reply:
@AnastasiaShemyakinskaya we can't. For patching, we need to modify CRD and kubernetes.Clientset only for standard Kubernetes objects